src/tests/ftest/dfuse/find.py

```diff
@@ -251,20 +225,18 @@ def _create_dir_forest(
             self.log.info("Populating: %s", path)
             prefix = "t{:05d}_".format(count)
             count += 1
-            remote_args = "{} {} {} {} {} {}".format(
-                path, height, subdirs, files_per_node, needles, prefix)
+            remote_args = ' '.join([path, height, subdirs, files_per_node, needles, prefix])
```

Comment:
files_per_node and possibly others here are ints, you could convert them on this line but it may be better to just leave them as strings throughout.